views/js/pciCreator/dev/mathEntryInteraction/runtime/scss/mathEntryInteraction.scss

```diff
@@ -36,86 +59,109 @@ $buttonTxt: blacken($base, 0.5);;
 $buttonTxtHover: white;
 $buttonFont: "Times New Roman", Times, serif;
 
-$buttonRadius: 4px; //todo: use mixin?
+$buttonRadius: 4px;
 
 $borderColor: blacken($base, 0.3);
 
-.qti-customInteraction {
-  .mathEntryInteraction {
-
-    .math-entry {
-      padding: 8px;
-    }
-
-    .math-entry-input {
-      width: 100%;
-      margin-top: 8px;
-    }
-
-    // Buttons
-    .math-entry-toolgroup {
-      margin: 8px 8px 8px 0;
-      display: inline-block;
+@mixin text-input() {
+    background-color: white;
+    border: 1px solid #ddd;
+}
 
-      border-radius: $buttonRadius;
-      border: 1px solid $borderColor;
+.qti-customInteraction {
+    .mathEntryInteraction {
+
+        .math-entry {
+            padding: 8px;
+
+            .toolbar {
+                margin-bottom: 8px;
+            }
+
+            .add-option {
```

Comment:
The add button style looks a strange for this usage.
What about setting a max-width to the button (~100px) so it looks like a button ? Have it at 100% feels like we are creating a new line with it.